### PR TITLE
[integration][idrac_secure_boot]- Added automation for export

### PR DESCRIPTION
**Pull request by Saksham-Nautiyal**
# Description
A few sentences describing the overall goals of the pull request's commits.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| |

# ISSUE TYPE

- Bugfix Pull Request
- Docs Pull Request
- Feature Pull Request
- Test Pull Request

##### OUTPUT

```paste below

```
##### ADDITIONAL INFORMATION

```paste below

```
# Checklist:

- [ ] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [ ] I have verified that new and existing unit tests pass locally with my changes
- [ ] I have not allowed coverage numbers to degenerate
- [ ] I have maintained at least 90% code coverage
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have maintained backward compatibility

### CODE REVIEW

**Comment by ansible-collections-svc** [comment]
Can one of the admins verify this patch?

File: tests/integration/targets/idrac_secure_boot/tests/export_all_certs.yaml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+      register: export_cert
+
+    - name: Verify - Export all policy certificate (normal mode)
+      ansible.builtin.assert:
+        that:
+          - not export_cert.changed
+          - export_cert.msg == "Successfully exported the SecureBoot
+           certificate."
+
+    # - name: Verify - Secure boot Log entries
```
Comment (ABHISHEK-SINHA10) [inline tests/integration/targets/idrac_secure_boot/tests/export_all_certs.yaml]: Remove the commented code if not needed, please check other files too.

File: tests/integration/targets/idrac_secure_boot/tests/export_database.yaml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+           certificate."
+
+    - name: Verify - Secure boot Log entries
+      ansible.builtin.include_tasks: _get_log_entries.yml
+      vars:
+        msg_id: "SWC9011"
+
+  module_defaults:
+    dellemc.openmanage.idrac_secure_boot:
+      idrac_ip: "{{ idrac_ip|ansible.utils.ipwrap }}"
```
Comment (ABHISHEK-SINHA10) [inline tests/integration/targets/idrac_secure_boot/tests/export_database.yaml]:
```suggestion
      idrac_ip: "{{ idrac_ip | ansible.utils.ipwrap }}"
```

Comment (ABHISHEK-SINHA10) [reply tests/integration/targets/idrac_secure_boot/tests/export_database.yaml]: Applies same to other files too.

File: tests/integration/targets/idrac_secure_boot/tests/export_all_certs.yaml (hunk 1 above)
Comment (Saksham-Nautiyal) [reply tests/integration/targets/idrac_secure_boot/tests/export_all_certs.yaml]: done

File: tests/integration/targets/idrac_secure_boot/tests/export_database.yaml (hunk 2 above)
Comment (Saksham-Nautiyal) [reply tests/integration/targets/idrac_secure_boot/tests/export_database.yaml]: done

**Review by ShivamSh3 — DISMISSED** [review]

**Review by singh-lovepreet1 — APPROVED** [review]

**Review by sachin-apa — APPROVED** [review]

**Pull request merged**